src/attrib.d

```diff
@@ -8,8 +8,34 @@
 
 module ddmd.attrib;
 
-import core.stdc.stdio, core.stdc.string;
-import ddmd.aggregate, ddmd.arraytypes, ddmd.cond, ddmd.dclass, ddmd.declaration, ddmd.dinterpret, ddmd.dmodule, ddmd.dscope, ddmd.dstruct, ddmd.dsymbol, ddmd.dtemplate, ddmd.errors, ddmd.expression, ddmd.func, ddmd.globals, ddmd.hdrgen, ddmd.id, ddmd.identifier, ddmd.mars, ddmd.mtype, ddmd.parse, ddmd.root.outbuffer, ddmd.root.rmem, ddmd.tokens, ddmd.utf, ddmd.visitor;
+import core.stdc.stdio;
+import core.stdc.string;
+import ddmd.aggregate;
+import ddmd.arraytypes;
+import ddmd.cond;
+import ddmd.dclass;
+import ddmd.declaration;
+import ddmd.dinterpret;
+import ddmd.dmodule;
+import ddmd.dscope;
+import ddmd.dstruct;
+import ddmd.dsymbol;
+import ddmd.dtemplate;
+import ddmd.errors;
+import ddmd.expression;
+import ddmd.func;
+import ddmd.globals;
+import ddmd.hdrgen;
+import ddmd.id;
+import ddmd.identifier;
+import ddmd.mars;
```

Comment:
Yeah, I agree.  That's why we have globals and errors, and some could probably go into root.  There are also probably a bunch of unnecessary imports around since I did them once by hand years ago and haven't updated them as things got cleaned up.